doc/api/console.markdown

```diff
@@ -93,7 +92,7 @@ Example:
       ;
     }
     console.timeEnd('100-elements');
-    // prints 100-elements: 262ms
+    // prints 100-elements: 225.438ms
```

Comment:
Bikeshedding: this belongs to the first commit «console: sub-millisecond accuracy for console.time», not to «doc: reword description of console.time».